src/views/Results.vue

```diff
@@ -216,10 +401,22 @@ export default {
 
 	beforeMount() {
 		this.loadFormResults()
+		this.loadLinkedFile()
 		SetWindowTitle(this.formTitle)
 	},
 
 	methods: {
+		async unlinkFile() {
+			await axios.post(generateOcsUrl('apps/forms/api/v2/submissions/unlink'), {
+				hash: this.form.hash,
+			})
+			await this.loadLinkedFile()
+		},
+		async loadLinkedFile() {
+			const response = await axios.get(generateOcsUrl(`apps/forms/api/v2/submissions/linkedFile/${this.form.hash}`))
```

Comment:
You can check the errors for loading the linked file here, e.g. error 404:

```suggestion
			try {
				const response = await axios.get(generateOcsUrl(`apps/forms/api/v2/submissions/linkedFile/${this.form.hash}`))
			} catch (e) {
				// handle the error, e.g.:
				// if (e?.response?.status === 404) { // not found...
			}
```

Reply:
done